app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,58 @@
+class PostsController < ApplicationController
+  before_action :set_post, only: [:index]
+
+  def index
+    @received_requests = current_user.friend_requests_received.pending
+    @posts = current_user.feed
+    @comments
+    # @comment = @post.comments
+  end
+
+  def create
+    @post = current_user.posts.build(post_params)
+    if @post.save
+      flash[:success] = "Post successfully created!!!"
+      redirect_to root_url
+    else
+      flash[:danger] = "Something prevented post from being created"
+      redirect_to root_path
+    end
+  end
+
+  def destroy
+    @post = Post.find(params[:id])
+    @post.destroy
+    flash[:success] = "Post successfully deleted!!!"
+    redirect_to posts_path
+  end
+
+
+
+  def post_comments
```

Comment:
Please see my comment in `users_controller.rb` about placing non-CRUD actions as public methods into our resource controllers.